WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationMainVM.kt

```diff
@@ -316,7 +316,9 @@ class SiteCreationMainVM @Inject constructor(
 
     fun onFreeSiteCreated(site: SiteModel) {
         siteCreationState = siteCreationState.copy(result = CreatedButNotFetched.NotInLocalDb(site))
-        wizardManager.showNextStep()
+        if (requireNotNull(siteCreationState.domain).isFree) {
```

Comment:
Every so tiny nitpick: we could use `checkNotNull` here, since the exception is more appropriate. Not a blocker, though :smile: 

```suggestion
        if (checkNotNull(siteCreationState.domain).isFree) {
```

Reply:
I agree, it's better tailored for this case to use `checkNotNull` ❤️ 
Thanks @mkevins 🙇🏻 !